extensions/amp-list/0.1/service/load-more-service.js

```diff
@@ -224,10 +224,10 @@ export class LoadMoreService {
     // If it's loading, then it's no longer failed or ended
     if (state) {
       this.getLoadMoreFailedElement().classList.toggle('amp-visible', false);
-      const loadMoreEndElement = this.getLoadMoreEndElement();
-      if (loadMoreEndElement) {
-        loadMoreEndElement.classList.toggle('amp-visible', false);
-      }
+    }
```

Comment:
This handles the case where you reach load-more-end, and then change the `[src]` attribute to an endpoint that still has a load-more.